### ISSUE TITLE
Attempting to convert player heads into EMC spams errors in console

### ISSUE THREAD

**Issue by Zynax1** [Sefiraat/EquivalencyTech#46]
## Description (Mandatory)
EquivalencyTech started spamming a bunch of errors in my server's console and some of the items (not sure if they are from this addon or another) stopped working.

## What caused the issue, what steps led up to the issue (Mandatory)
I'm not exactly sure how it was caused, all I know is that the errors started when a player called Pop_user2 joined the server
Edit: The errors seem to spam when you attempt to convert a player head into EMC

## Log of error, if applicable
https://zerobin.net/?f7ab1c18628a9a80#mTmo0bVBgI1GL6xsyCx4VMJ0128L1yVCGo5ThIEcCcU=

## Server Type and version (Spigot/Paper etc.)
Paper 1.18.2 
## Minecraft version
1.18.2

## Slimefun Setup

[14:10:55 INFO]: This Server uses the following setup of Slimefun:
Paper git-Paper-267 (MC: 1.18.2)
Slimefun DEV - 1008 (git 020094e0)
Metrics-Module #28
Java 17
Installed Addons: (14)
  SlimefunOreChunks vDEV - 15 (git 2520cb23)
  AlchimiaVitae vDEV - 19 (git 24ffb2a5)
  EcoPower vDEV - 5 (git 5c1ecd68)
  DynaTech vDEV - 160 (git e1075a1c)
  ExtraGear vDEV - 13 (git c62047e2)
  ExoticGarden vDEV - 78 (git 7f9a5f63)
  InfinityExpansion vDEV - 137 (git 4b326e08)
  LiteXpansion vDEV - 174 (git e36d509d)
  SoulJars vDEV - 23 (git b8358a1f)
  SlimyTreeTaps vDEV - 20 (git 963f0383)
  EquivalencyTech v${project.version}
  DankTech v1.2.11
  PrivateStorage vDEV - 18 (git 5832db21)
  SlimeTinker vDEV - 72 (git bb2ef153)

**Comment by ifloppy** [comment]
Your player head is not available in the EquivalencyTech

**Comment by Zynax1** [comment]
Yeah I know that heads don't have an emc value, it's just a pain that it spams errors in the console when attempted.
